src/main/java/org/spongepowered/api/profile/property/ProfileProperty.java

```diff
@@ -66,6 +74,16 @@ static ProfileProperty of(String name, String value, @Nullable String signature)
     }
 
     /**
+     * Creates a new textures property.
+     *
+     * @param value The value of the property
```

Comment:
You should add a note that it's base64 encoded, and maybe link to a description of the format.